### DISCUSSION TITLE
How to edit the "menu headings" in "mega menu"?

### DISCUSSION THREAD

**Discussion by MagnusNW** [templaza/astroid-framework#477] [General]
I have actually 2 questions regarding the design of mega menu.

1. Where do I change the "menu heading" style? see attached file "mega menu"
2. How to create "dividers" so it look as in the tutorial video? see attached file "mega menu2"
![Mega menu](https://github.com/templaza/astroid-framework/assets/152264716/eb800e13-ec45-424a-a417-ad9b0d5faa73)
![Mega menu 2](https://github.com/templaza/astroid-framework/assets/152264716/39be3ef2-fecb-4b9e-b606-08c3f78594ac)

**Comment by Chacapamac** [comment]
@MagnusNW 
The only way I see is by using custom CSS to override the default styling.

**Reply by MagnusNW** [reply]
Yeah ok, I suspected that too.
